source/api/index.md

```diff
@@ -42,6 +42,17 @@ __Translation Note__<br/>
 Please note that the IIIF community does not guarantee the accuracy of any translation. They are linked to for information purposes only, and any discrepancies with the specifications are unintentional. The English versions of the specifications linked above are the definitive versions.
 {: .alert}
 
+## Community Cookbook
+
+The [cookbook][cookbook] gathers together examples of how to create IIIF Presentation API assets, in order to:
```

Comment:
I would propose that the description also includes a note that the cookbook is a constant work in progress of the community. E.g. the landing page is quite long, with lots of non-links, incomprehensible (to the casual viewer) numbers, and so on. Then the referenced pages have placeholder links in []s that look like mistakes, and so on.

Cookbook link should go into the _links include rather than the page directly.

Reply:
Hopefully resolved now